OpenSees.prb

```diff
@@ -224,54 +224,173 @@ QUESTION: _#CB#(_)
 VALUE: Select output and specify layer numbers for up to three layers (A,B,C)
 QUESTION: _#CB#(_)
 VALUE:
+QUESTION: _#CB#(_)
+VALUE:
 QUESTION: Stresses_in_layer_A#CB#(1,0)
 VALUE: 0
-DEPENDENCIES: (0,HIDE,Stress_layer_A,#CURRENT#) (1,RESTORE,Stress_layer_A,#CURRENT#)
-QUESTION: Stress_layer_A
+DEPENDENCIES: (0,HIDE,Stress_layer_A,ConcreteTop,HIDE,Layer_Material_A,#CURRENT#,HIDE,Layer_Number_A,#CURRENT#) 
+DEPENDENCIES: (1,RESTORE,Stress_layer_A,#CURRENT#,SET,Layer_Material_A,#CURRENT#,SET,Layer_Number_A,#CURRENT#)
+QUESTION: Stress_layer_A#CB#(SteelTopLongitudinal,SteelTopTransverse,SteelBotLongitudinal,SteelBotTransverse,BottomDeck,ConcreteTop,ConcreteBottom,CustomLayer)
```

Comment:
These options are all very useful. However, when working with a big model, having recorders for only 3 layers is painfully limiting to work with. I suggest the following additional recorder options:

All steel: in your bas file you make it that if this particular option is selected, then all steel options are selected and output is recorded for all reinforcement and decking layers. 
Top & Bottom concrete: both top and bottom concrete in one option.

These two additional options would be what we need > 80% of the time.

Reply:
Great! I will update the progress soon with the suggested changes